packages/components/src/Portal/Portal.tsx

```diff
@@ -92,7 +93,14 @@ export const Portal = forwardRef(
 
 Portal.displayName = 'Portal'
 
+/**
+ * InvisiBox is mounted outside of StyleDefender / ComponentsProvider
+ * DOM-output so it re-injects `styleDefenderCSS` to do a light-weight
+ * "CSS reset"
+ */
 const InvisiBox = styled.div<PortalPlacementProps>`
+  ${styleDefenderCSS}
```

Comment:
The white background blocks out what's underneath.
```suggestion
  ${styleDefenderCSS}
  background: transparent;
```
![de9e81f8-020a-4965-8c22-a07b8180e10b](https://user-images.githubusercontent.com/53451193/115077059-4beb2100-9eb2-11eb-8c5a-aece3f4c0eae.gif)

Reply:
But even with that background style change it seems like somethings off with the `Popover` click outside behavior on `/?path=/story/popover--popover-focus-trap` – with "Cancel Click Outside" off, the "Click me with the popover open" button should trigger an alert. I can dig into it a bit more next week.

Reply:
Please ignore that last comment. It's working normally with or without the background. I must've been trippin.

Reply:
Move background into `StyleDefender` component instead of share CSS block to address background overlay issue. Realized that I had tested in Dialog/Drawer but not Popover 🤦